diff --git a/toolkit/content/aboutSupport.xhtml b/toolkit/content/aboutSupport.xhtml
index 296259cd0360a403e8659e22d0c08e968529a38a..bde1ee8d0a05e6ce2eb3ff8ff8a19ba28c20af98 100644
--- a/toolkit/content/aboutSupport.xhtml
+++ b/toolkit/content/aboutSupport.xhtml
@@ -10,6 +10,7 @@
 
 <html xmlns="http://www.w3.org/1999/xhtml">
   <head>
+    <script src="chrome://browser/content/zenThemeModifier.js"></script>
     <meta http-equiv="Content-Security-Policy" content="default-src chrome:; object-src 'none'" />
     <meta name="color-scheme" content="light dark" />
     <title data-l10n-id="page-title"/>
