diff --git a/browser/components/preferences/main.js b/browser/components/preferences/main.js
index 8f6f78419a0876ebe6d0967a88ef03e464dbbd23..9339085014b797033c4ee1ed2e40f2506d1e9797 100644
--- a/browser/components/preferences/main.js
+++ b/browser/components/preferences/main.js
@@ -218,7 +218,7 @@ function getBundleForLocales(newLocales) {
     ])
   );
   return new Localization(
-    ["browser/preferences/preferences.ftl", "branding/brand.ftl"],
+    ["browser/preferences/preferences.ftl", "branding/brand.ftl", "browser/preferences/zen-preferences.ftl"],
     false,
     undefined,
     locales
