diff --git a/browser/components/preferences/main.inc.xhtml b/browser/components/preferences/main.inc.xhtml
index f9129fb171362e81369056923bf25acbb87d2ad8..924bcd2cd66a5dfcf9c45aac0c63cd0257c21a2c 100644
--- a/browser/components/preferences/main.inc.xhtml
+++ b/browser/components/preferences/main.inc.xhtml
@@ -29,6 +29,9 @@
   <vbox id="startupPageBox">
     <checkbox id="browserRestoreSession"
               data-l10n-id="startup-restore-windows-and-tabs"/>
+    <checkbox id="zenWorkspaceContinueWhereLeftOff"
+            data-l10n-id="zen-workspace-continue-where-left-off"
+            preference="zen.workspaces.continue-where-left-off"/>
 #ifdef XP_WIN
     <hbox id="windowsLaunchOnLoginBox" align="center" hidden="true">
       <checkbox id="windowsLaunchOnLogin"
@@ -186,6 +189,7 @@
 </groupbox>
 
 <!-- Browser layout -->
+#if 0
 <groupbox data-category="paneGeneral"
           data-subcategory="layout"
           hidden="true">
@@ -201,6 +205,7 @@
             preference="sidebar.revamp"/>
   <description class="indent" data-l10n-id="browser-layout-show-sidebar-desc"></description>
 </groupbox>
+#endif
 
 <hbox id="languageAndAppearanceCategory"
       class="subcategory"
@@ -421,6 +426,7 @@
               languages-customize-add.label,
             " />
   </hbox>
+  <label class="zenLanguageWarning-label"><html:b data-l10n-id="zen-warning-language"/></label>
 
   <checkbox id="useSystemLocale" hidden="true"
             data-l10n-id="use-system-locale"
