diff --git a/toolkit/components/prompts/content/commonDialog.xhtml b/toolkit/components/prompts/content/commonDialog.xhtml
index c47761ec958a11870e55294fe9c6de8f3afbda2f..5e0313864c67a774874c215fd043b9b8af39d8cb 100644
--- a/toolkit/components/prompts/content/commonDialog.xhtml
+++ b/toolkit/components/prompts/content/commonDialog.xhtml
@@ -30,6 +30,7 @@
 
       <html:link rel="localization" href="branding/brand.ftl" />
       <html:link rel="localization" href="toolkit/global/commonDialog.ftl" />
+      <script src="chrome://browser/content/zenThemeModifier.js"></script>
     </linkset>
     <script src="chrome://global/content/adjustableTitle.js" />
     <script src="chrome://global/content/commonDialog.js" />
