diff --git a/layout/style/test/ListCSSProperties.cpp b/layout/style/test/ListCSSProperties.cpp
index b6a881b037a3720991f8097903e859dfc72012c1..8917a8cc23b97e37b4accaaf0d6c2876b36594af 100644
--- a/layout/style/test/ListCSSProperties.cpp
+++ b/layout/style/test/ListCSSProperties.cpp
@@ -108,6 +108,7 @@ const char* gInaccessibleProperties[] = {
     "-moz-window-opacity",                // chrome-only internal properties
     "-moz-window-transform",              // chrome-only internal properties
     "-moz-window-shadow",                 // chrome-only internal properties
+    "-zen-window-transform-origin",       // chrome-only internal properties
 };
 
 inline int is_inaccessible(const char* aPropName) {
