diff --git a/browser/base/content/aboutDialog.js b/browser/base/content/aboutDialog.js
index f6e1391baf12abb91c85a95107bb3923118746c0..76c7b75a4e29056110f1631a50047c4ddd8b1f4a 100644
--- a/browser/base/content/aboutDialog.js
+++ b/browser/base/content/aboutDialog.js
@@ -52,7 +52,7 @@ function init() {
   ]);
   let versionIdKey = "base";
   let versionAttributes = {
-    version: AppConstants.MOZ_APP_VERSION_DISPLAY,
+    version: `${AppConstants.MOZ_APP_VERSION_DISPLAY} (Firefox ${AppConstants.ZEN_FIREFOX_VERSION})`,
   };
 
   let arch = Services.sysinfo.get("arch");
@@ -64,7 +64,7 @@ function init() {
   }
 
   let version = Services.appinfo.version;
-  if (/a\d+$/.test(version)) {
+  if (version.includes("t")) {
     versionIdKey += "-nightly";
     let buildID = Services.appinfo.appBuildID;
     let year = buildID.slice(0, 4);
