diff --git a/browser/base/content/browser.xhtml b/browser/base/content/browser.xhtml
index 51a25aaa5558e6e17246d54a7ed95d5ddf3ecdab..b49984c8711fc9f5f19f0cf6ecca07a8cca0d125 100644
--- a/browser/base/content/browser.xhtml
+++ b/browser/base/content/browser.xhtml
@@ -26,6 +26,7 @@
         sizemode="normal"
         retargetdocumentfocus="urlbar-input"
         scrolling="false"
+        zen-before-loaded="true"
         persist="screenX screenY width height sizemode"
         data-l10n-sync="true">
 <head>
@@ -104,8 +105,11 @@
 
   <title data-l10n-id="browser-main-window-default-title"></title>
 
+#include zen-preloaded.inc.xhtml
+
   <script src="chrome://browser/content/global-scripts.js"></script>
   <script src="chrome://browser/content/browser-main.js"></script>
+#include zen-assets.inc.xhtml
 </head>
 <html:body xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul">
 # All sets except for popupsets (commands, keys, and stringbundles)
@@ -127,9 +131,12 @@
     </vbox>
   </html:template>
 
+<hbox id="zen-main-app-wrapper" flex="1" persist="zen-compact-mode">
+  <vbox id="zen-toast-container"></vbox>
 #include navigator-toolbox.inc.xhtml
 
 #include browser-box.inc.xhtml
+</hbox>
 
   <html:template id="customizationPanel">
     <box id="customization-container" flex="1" hidden="true"><![CDATA[
