diff --git a/browser/base/content/navigator-toolbox.inc.xhtml b/browser/base/content/navigator-toolbox.inc.xhtml
index feb9e4ef7e872b4cfbf0d55f24a34db275951c66..3ce9b9919ff736885b8aa8943bbbce3c5d0f33a6 100644
--- a/browser/base/content/navigator-toolbox.inc.xhtml
+++ b/browser/base/content/navigator-toolbox.inc.xhtml
@@ -2,7 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-<toolbox id="navigator-toolbox" class="browser-toolbox-background">
+<toolbox id="navigator-toolbox" class="browser-toolbox-background" persist="width style">
   <script src="chrome://browser/content/navigator-toolbox.js" />
 
   <!-- Menu -->
@@ -17,9 +17,8 @@
 #include browser-menubar.inc
     </toolbaritem>
     <spacer flex="1" skipintoolbarset="true" style="order: 1000;"/>
-#include titlebar-items.inc.xhtml
   </toolbar>
-
+<hbox id="titlebar">
   <toolbar id="TabsToolbar"
            class="browser-toolbar browser-titlebar"
            fullscreentoolbar="true"
@@ -32,7 +31,7 @@
 
     <hbox class="titlebar-spacer" type="pre-tabs"/>
 
-    <hbox flex="1" align="end" class="toolbar-items">
+    <hbox flex="1" align="start" class="toolbar-items">
       <toolbartabstop/>
       <hbox id="TabsToolbar-customization-target" flex="1">
         <toolbarbutton id="firefox-view-button"
@@ -40,6 +39,7 @@
                        data-l10n-id="toolbar-button-firefox-view-2"
                        role="button"
                        aria-pressed="false"
+                       hidden="true"
                        cui-areatype="toolbar"
                        removable="true"/>
 
@@ -50,6 +50,10 @@
               tooltip="tabbrowser-tab-tooltip"
               orient="horizontal"
               stopwatchid="FX_TAB_CLICK_MS">
+<html:div id="zen-essentials-container" skipintoolbarset="true"></html:div>
+<hbox id="zen-current-workspace-indicator-container"></hbox>
+<html:div id="zen-tabs-wrapper">
+<html:div id="zen-browser-tabs-container">
           <hbox class="tab-drop-indicator" hidden="true"/>
 # If the name (tabbrowser-arrowscrollbox) or structure of this changes
 # significantly, there is an optimization in
@@ -75,6 +79,8 @@
                               tooltip="dynamic-shortcut-tooltip"
                               data-l10n-id="tabs-toolbar-new-tab"/>
           <html:span id="tabbrowser-tab-a11y-desc" hidden="true"/>
+</html:div>
+</html:div>
         </tabs>
 
         <toolbarbutton id="new-tab-button"
@@ -100,10 +106,10 @@
 #include private-browsing-indicator.inc.xhtml
     <toolbarbutton class="content-analysis-indicator toolbarbutton-1 content-analysis-indicator-icon"/>
 
-#include titlebar-items.inc.xhtml
+#include zen-sidebar-icons.inc.xhtml
 
   </toolbar>
-
+</hbox>
   <toolbar id="nav-bar"
            class="browser-toolbar chromeclass-location"
            data-l10n-id="navbar-accessible"
@@ -489,10 +495,10 @@
                      consumeanchor="PanelUI-button"
                      data-l10n-id="appmenu-menu-button-closed2"/>
     </toolbaritem>
+#include titlebar-items.inc.xhtml
     <hbox class="titlebar-spacer" type="post-tabs"/>
 #include private-browsing-indicator.inc.xhtml
     <toolbarbutton class="content-analysis-indicator toolbarbutton-1 content-analysis-indicator-icon"/>
-#include titlebar-items.inc.xhtml
   </toolbar>
 
   <toolbar id="PersonalToolbar"
