diff --git a/browser/base/content/nsContextMenu.sys.mjs b/browser/base/content/nsContextMenu.sys.mjs
index 3bd0ee8af8b696161d18fd2a978f9e25972ed3ad..2f0763dbde2250f625e9a87ab88910b4421c36ca 100644
--- a/browser/base/content/nsContextMenu.sys.mjs
+++ b/browser/base/content/nsContextMenu.sys.mjs
@@ -1111,6 +1111,13 @@ export class nsContextMenu {
         !this.isSecureAboutPage()
     );
 
+    this.showItem(
+      "context-zenAddToWebPanel",
+      this.onLink && !this.onMailtoLink && !this.onTelLink
+    );
+
+    this.showItem("context-zenSplitLink", this.onLink && !this.onMailtoLink && !this.onTelLink);
+
     let canNotStrip =
       lazy.STRIP_ON_SHARE_CAN_DISABLE && !this.#canStripParams();
 
