diff --git a/browser/base/content/nsContextMenu.sys.mjs b/browser/base/content/nsContextMenu.sys.mjs
index 94d3c2bb401c2e4246340a6c2d63c9d3561ff5e1..62c599f78748e708c8f36c05935822e8808996b5 100644
--- a/browser/base/content/nsContextMenu.sys.mjs
+++ b/browser/base/content/nsContextMenu.sys.mjs
@@ -1109,6 +1109,13 @@ export class nsContextMenu {
         !this.isSecureAboutPage()
     );
 
+    this.showItem(
+      "context-zenAddToWebPanel",
+      this.onLink && !this.onMailtoLink && !this.onTelLink
+    );
+
+    this.showItem("context-zenSplitLink", this.onLink && !this.onMailtoLink && !this.onTelLink);
+
     let canNotStrip =
       lazy.STRIP_ON_SHARE_CAN_DISABLE && !this.#canStripParams();
 
