diff --git a/browser/themes/BuiltInThemeConfig.sys.mjs b/browser/themes/BuiltInThemeConfig.sys.mjs
index 081187147736f56abfe5866a69e00ac13a8dd940..9a980fe858c1a77af7634037aebc332d237e9c11 100644
--- a/browser/themes/BuiltInThemeConfig.sys.mjs
+++ b/browser/themes/BuiltInThemeConfig.sys.mjs
@@ -33,13 +33,6 @@ export const BuiltInThemeConfig = new Map([
       path: "resource://builtin-themes/dark/",
     },
   ],
-  [
-    "firefox-alpenglow@mozilla.org",
-    {
-      version: "1.5",
-      path: "resource://builtin-themes/alpenglow/",
-    },
-  ],
   [
     "2022red-colorway@mozilla.org",
     {
