diff --git a/browser/themes/BuiltInThemes.sys.mjs b/browser/themes/BuiltInThemes.sys.mjs
index cc7f6cf93e04d339fb79a028b1e5ac6e79eae9c3..85e0d653f9a6d577bdea2a1c9d53018dd45684fa 100644
--- a/browser/themes/BuiltInThemes.sys.mjs
+++ b/browser/themes/BuiltInThemes.sys.mjs
@@ -74,7 +74,7 @@ class _BuiltInThemes {
   maybeInstallActiveBuiltInTheme() {
     const activeThemeID = Services.prefs.getStringPref(
       kActiveThemePref,
-      "default-theme@mozilla.org"
+      "firefox-compact-dark@mozilla.org"
     );
     let activeBuiltInTheme = this.builtInThemeMap.get(activeThemeID);
 
@@ -166,7 +166,7 @@ class _BuiltInThemes {
   async _uninstallExpiredThemes() {
     const activeThemeID = Services.prefs.getStringPref(
       kActiveThemePref,
-      "default-theme@mozilla.org"
+      "firefox-compact-dark@mozilla.org"
     );
     const now = new Date();
     const expiredThemes = Array.from(this.builtInThemeMap.entries()).filter(
