diff --git a/browser/components/protections/content/protections.mjs b/browser/components/protections/content/protections.mjs
index 597b6d93ec..d813ef24bd 100644
--- a/browser/components/protections/content/protections.mjs
+++ b/browser/components/protections/content/protections.mjs
@@ -391,6 +391,10 @@ document.addEventListener("DOMContentLoaded", () => {
   }).then(createGraph);
 
   let exitIcon = document.querySelector("#mobile-hanger .exit-icon");
+  
+  // Enforce the mobile app promotion to be hidden on load.
+  RPMSetPref("browser.contentblocking.report.show_mobile_app", false);
+
   // hide the mobile promotion and keep hidden with a pref.
   exitIcon.addEventListener("click", () => {
     RPMSetPref("browser.contentblocking.report.show_mobile_app", false);
