diff --git a/toolkit/modules/LightweightThemeConsumer.sys.mjs b/toolkit/modules/LightweightThemeConsumer.sys.mjs
index d8c79cbae4de3ebfc5dc62f3597c7be4965a5429..ba1a44a0be5b5837467cfc84074d4869aee6aa62 100644
--- a/toolkit/modules/LightweightThemeConsumer.sys.mjs
+++ b/toolkit/modules/LightweightThemeConsumer.sys.mjs
@@ -31,7 +31,7 @@ const toolkitVariableMap = [
       lwtProperty: "accentcolor",
       processColor(rgbaChannels) {
         if (!rgbaChannels || rgbaChannels.a == 0) {
-          return "white";
+          return "var(--zen-colors-primary)";
         }
         // Remove the alpha channel
         const { r, g, b } = rgbaChannels;
@@ -339,6 +339,7 @@ LightweightThemeConsumer.prototype = {
       theme.additionalBackgrounds
     );
     let _processedColors = _setProperties(root, hasTheme, theme);
+    this._zenSetAccentColor(root);
 
     if (hasTheme) {
       if (updateGlobalThemeData) {
@@ -431,6 +432,14 @@ LightweightThemeConsumer.prototype = {
       this._lastExperimentData.stylesheet = stylesheet;
     }
   },
+
+  _zenSetAccentColor(root) {
+    const themeAccent = Services.prefs.getStringPref(
+      "zen.theme.accent-color",
+      "#0b57d0"
+    );
+    root.style.setProperty("--zen-primary-color", themeAccent);
+  },
 };
 
 function _getContentProperties(doc, hasTheme, data) {
