diff --git a/browser/branding/branding-common.mozbuild b/browser/branding/branding-common.mozbuild
index a38663a3dc1ebe0c0817945efc2f72018bc1b3c7..8d9d314a718018ae074f1fb89d93dbfdcc51f36f 100644
--- a/browser/branding/branding-common.mozbuild
+++ b/browser/branding/branding-common.mozbuild
@@ -12,7 +12,7 @@ def FirefoxBranding():
             "pref/firefox-branding.js",
         ]
     else:
-        JS_PREFERENCE_FILES += [
+        JS_PREFERENCE_PP_FILES += [
             "pref/firefox-branding.js",
         ]
 
