diff --git a/browser/components/places/jar.mn b/browser/components/places/jar.mn
index e12c6bbf449f062fd22ef260909b539857f52c6a..2dcc143109de015e4c9d486b83119ae1122d2e71 100644
--- a/browser/components/places/jar.mn
+++ b/browser/components/places/jar.mn
@@ -25,3 +25,5 @@ browser.jar:
     content/browser/places/interactionsViewer.css        (metadataViewer/interactionsViewer.css)
     content/browser/places/interactionsViewer.html       (metadataViewer/interactionsViewer.html)
     content/browser/places/interactionsViewer.js         (metadataViewer/interactionsViewer.js)
+    content/browser/places/zenNewWebPanel.xhtml      (content/zenNewWebPanel.xhtml)
+    content/browser/places/zenNewWebPanel.js         (content/zenNewWebPanel.js)
