diff --git a/toolkit/actors/AudioPlaybackParent.sys.mjs b/toolkit/actors/AudioPlaybackParent.sys.mjs
index db682fd90b2bb5330497d2cf2158ff4cac6bbc47..4e6f891275d489418b8ea58a10345a1baa3a554c 100644
--- a/toolkit/actors/AudioPlaybackParent.sys.mjs
+++ b/toolkit/actors/AudioPlaybackParent.sys.mjs
@@ -14,10 +14,12 @@ export class AudioPlaybackParent extends JSWindowActorParent {
     switch (aMessage.name) {
       case "AudioPlayback:Start":
         this._hasAudioPlayback = true;
+        browser.ownerGlobal.gZenMediaController.activateMediaControls(this.browsingContext.mediaController, browser);
         browser.audioPlaybackStarted();
         break;
       case "AudioPlayback:Stop":
         this._hasAudioPlayback = false;
+        browser.ownerGlobal.gZenMediaController.updateMuteState();
         browser.audioPlaybackStopped();
         break;
       case "AudioPlayback:ActiveMediaBlockStart":
