diff --git a/toolkit/actors/SelectParent.sys.mjs b/toolkit/actors/SelectParent.sys.mjs
index 3e1d4b73b3d84a550222827230917c10ce9b8448..1d2ada14a8ca2ee45a2abdc118068a708816365c 100644
--- a/toolkit/actors/SelectParent.sys.mjs
+++ b/toolkit/actors/SelectParent.sys.mjs
@@ -103,7 +103,7 @@ export var SelectParentHelper = {
     }
 
     menupopup.setAttribute("style", "");
-    menupopup.style.colorScheme = isDarkBackground ? "dark" : "light";
+    //menupopup.style.colorScheme = isDarkBackground ? "dark" : "light";
     menupopup.style.direction = selectStyle.direction;
 
     stylesheet = doc.createElementNS("http://www.w3.org/1999/xhtml", "style");
