diff --git a/browser/extensions/newtab/lib/AboutPreferences.sys.mjs b/browser/extensions/newtab/lib/AboutPreferences.sys.mjs
index a325becabddda511204fc5bd2022edc332cdde7c..6bb29b21f1e6c1d6ea3d38d50f01148e0a51f0d7 100644
--- a/browser/extensions/newtab/lib/AboutPreferences.sys.mjs
+++ b/browser/extensions/newtab/lib/AboutPreferences.sys.mjs
@@ -52,7 +52,7 @@ const PREFS_BEFORE_SECTIONS = () => [
       titleString: "home-prefs-shortcuts-header",
       descString: "home-prefs-shortcuts-description",
       get nestedPrefs() {
-        return Services.prefs.getBoolPref("browser.topsites.useRemoteSetting")
+        return Services.prefs.getBoolPref("browser.topsites.useRemoteSetting") && false
           ? [
               {
                 name: "showSponsoredTopSites",
