diff --git a/dom/chrome-webidl/MediaController.webidl b/dom/chrome-webidl/MediaController.webidl
index 20f416d1c3b41798e0f90bbac5db40ed2a4ab000..06cb4c847fcfba964eeb93089613e293dc10bd87 100644
--- a/dom/chrome-webidl/MediaController.webidl
+++ b/dom/chrome-webidl/MediaController.webidl
@@ -20,6 +20,12 @@ enum MediaControlKey {
   "stop",
 };
 
+dictionary MediaControllerPositionState {
+  required double duration;
+  required double playbackRate;
+  required double position;
+};
+
 /**
  * MediaController is used to control media playback for a tab, and each tab
  * would only have one media controller, which can be accessed from the
@@ -36,6 +42,9 @@ interface MediaController : EventTarget {
   [Throws]
   MediaMetadataInit getMetadata();
 
+  [Throws]
+  MediaControllerPositionState getPositionState();
+
   [Frozen, Cached, Pure]
   readonly attribute sequence<MediaControlKey> supportedKeys;
 
