diff --git a/toolkit/components/pictureinpicture/PictureInPicture.sys.mjs b/toolkit/components/pictureinpicture/PictureInPicture.sys.mjs
index 5da0404b2672ba8cce7bcf808bf2373474776654..6a22fce1212bd6ac6bdd4962b8247c3db956f0eb 100644
--- a/toolkit/components/pictureinpicture/PictureInPicture.sys.mjs
+++ b/toolkit/components/pictureinpicture/PictureInPicture.sys.mjs
@@ -126,6 +126,9 @@ export class PictureInPictureToggleParent extends JSWindowActorParent {
         if (browser.ownerGlobal.gBrowser.selectedBrowser == browser) {
           break;
         }
+        if (browser.audioMuted) {
+          break;
+        }
         let actor = browsingContext.currentWindowGlobal.getActor(
           "PictureInPictureLauncher"
         );
@@ -494,7 +497,7 @@ export var PictureInPicture = {
     // focus the tab's window
     tab.ownerGlobal.focus();
 
-    gBrowser.selectedTab = tab;
+    browser?.ownerGlobal?.ZenWorkspaces.switchIfNeeded(browser);
     await this.closeSinglePipWindow({ reason: "Unpip", actorRef: pipActor });
   },
 
