diff --git a/toolkit/components/pictureinpicture/PictureInPicture.sys.mjs b/toolkit/components/pictureinpicture/PictureInPicture.sys.mjs
index 5da0404b2672ba8cce7bcf808bf2373474776654..1405c53a927a64ecda8255d78d9aa5ddf8806685 100644
--- a/toolkit/components/pictureinpicture/PictureInPicture.sys.mjs
+++ b/toolkit/components/pictureinpicture/PictureInPicture.sys.mjs
@@ -494,7 +494,7 @@ export var PictureInPicture = {
     // focus the tab's window
     tab.ownerGlobal.focus();
 
-    gBrowser.selectedTab = tab;
+    browser?.ownerGlobal?.ZenWorkspaces.switchIfNeeded(browser);
     await this.closeSinglePipWindow({ reason: "Unpip", actorRef: pipActor });
   },
 
@@ -623,6 +623,7 @@ export var PictureInPicture = {
       pipToggle.hidden = true;
     }
 
+    win.gZenMediaController.updatePipEligibility(browser, !pipToggle.hidden);
     let browserHasPip = !!this.browserWeakMap.get(browser);
     if (browserHasPip) {
       this.setUrlbarPipIconActive(browser.ownerGlobal);
