diff --git a/browser/components/sidebar/browser-sidebar.js b/browser/components/sidebar/browser-sidebar.js
index 371190c058fa1eed91dc91b58608c934100e3a1b..440fe6fb403c3e78b3bfe68fb747d000b4a27521 100644
--- a/browser/components/sidebar/browser-sidebar.js
+++ b/browser/components/sidebar/browser-sidebar.js
@@ -718,7 +718,7 @@ var SidebarController = {
         }
       });
     } else {
-      let browser = document.getElementById("browser");
+      let browser = document.getElementById("tabbrowser-tabbox");
       [...browser.children].forEach((node, i) => {
         node.style.order = i + 1;
       });
