diff --git a/Cargo.toml b/Cargo.toml
index d3d529b487f80d4b4f3bfa13cdd3f62946352db8..b35a9e0e3339d48d1cd0cab7b22d030f330e40d1 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -217,6 +217,9 @@ rure = { path = "third_party/rust/rure" }
 # Patch `plist` to work with `indexmap` 2.*
 plist = { path = "third_party/rust/plist" }
 
+# Patch mime_guess to add missing mime types
+mime_guess = { path = "third_party/rust/mime_guess" }
+
 # Patch `unicode-width` 0.1.* to 0.2.
 unicode-width = { path = "build/rust/unicode-width" }
 
